src_c/line.c

```diff
@@ -490,6 +490,21 @@ pg_line_flip_ip(pgLineObject *self, PyObject *_null)
     Py_RETURN_NONE;
 }
 
+static PyObject *
+pg_line_as_circle(pgLineObject *self, PyObject *_null)
+{
+    pgCircleObject *circle_obj =
+        (pgCircleObject *)pgCircle_Type.tp_new(&pgCircle_Type, NULL, NULL);
+
+    if (circle_obj) {
+        circle_obj->circle.x = (self->line.x1 + self->line.x2) / 2;
+        circle_obj->circle.y = (self->line.y1 + self->line.y2) / 2;
+        circle_obj->circle.r = pgLine_Length(&self->line) / 2;
+    }
+
+    return (PyObject *)circle_obj;
```

Comment:
I think this can be reduced to:
```C
return pgCircle_New3((self->line.x1 + self->line.x2) / 2,
                     (self->line.y1 + self->line.y2) / 2, pgLine_Length(&self->line) / 2);
```

Reply:
![image](https://user-images.githubusercontent.com/44055981/219002377-832fa184-c278-45a9-b164-e05fcab8f28b.png)
`pgCircle_New3` is inaccessible from `line.c`.